src/test/java/org/folio/list/service/ListRefreshServiceTest.java

```diff
@@ -54,6 +59,11 @@ class ListRefreshServiceTest {
   @Mock
   private EntityManagerFlushService entityManagerFlushService;
 
+  @BeforeEach
+  void setup() {
+    ReflectionTestUtils.setField(listRefreshService, "getQueryTimeoutMinutes", 10);
+  }
```

Comment:
Super annoying: @Value annotations don't do anything in tests unless you do a full @SpringBootTest or jump through some config hoops. But tests fail when the timeout isn't set, so I just took the lazy approach and set the value manually with reflection